Makefile

```diff
@@ -5,45 +5,45 @@ GO_BUILD_BINDIR :=./bin
 GIT_COMMIT := $(or $(SOURCE_GIT_COMMIT),$(shell git rev-parse --short HEAD))
 GIT_TAG :="$(shell git tag | sort -V | tail -1)"
 
-GO_LD_EXTRAFLAGS :=-X github.com/uor-framework/client/cli.version="$(shell git tag | sort -V | tail -1)" \
-				   -X github.com/uor-framework/client/cli.buildData="dev" \
-				   -X github.com/uor-framework/client/cli.commit="$(GIT_COMMIT)" \
-				   -X github.com/uor-framework/client/cli.buildDate="$(shell date -u +'%Y-%m-%dT%H:%M:%SZ')"
+GO_LD_EXTRAFLAGS :=-X github.com/uor-framework/uor-client-go/cli.version="$(shell git tag | sort -V | tail -1)" \
+				   -X github.com/uor-framework/uor-client-go/cli.buildData="dev" \
+				   -X github.com/uor-framework/uor-client-go/cli.commit="$(GIT_COMMIT)" \
+				   -X github.com/uor-framework/uor-client-go/cli.buildDate="$(shell date -u +'%Y-%m-%dT%H:%M:%SZ')"
 
 build: prep-build-dir
-	$(GO) build -o $(GO_BUILD_BINDIR)/client  -ldflags="$(GO_LD_EXTRAFLAGS)" $(GO_BUILD_PACKAGES)
+	$(GO) build -o $(GO_BUILD_BINDIR)/uor-client-go  -ldflags="$(GO_LD_EXTRAFLAGS)" $(GO_BUILD_PACKAGES)
```

Comment:
Do we want to keep the binary name `client` for now?

Reply:
Client is so generic. So we would want to align to just the repo name or a more long term name of the client

Reply:
I am good with just using the repo name for now until we determine the long-term name.

Reply:
Sounds good. As soon as we want to release, we should update website content (I can do that)

Reply:
I'm interested in this conversation.

At different points, we've discussed this repo serving as a reference implementation and not a formal "product" producing repository. We have also discussed taking on a [kaiju](https://en.wikipedia.org/wiki/Kaiju) based branding theme and the client becoming something like `kaictl`.

Should we go ahead and document some formal outcomes to make a decision on this?